components/LinkMarkdown.tsx

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import Link from 'next/link';
 
 const LinkMarkdown = (props: { href: string; children: string }): JSX.Element => {
-    return props.href.startsWith('#') ? (
+    return !props.href ? (
+        <a>{props.children}</a>
+    ) : props.href.startsWith('#') || props.href.startsWith('http') ? (
```

Comment:
```suggestion
    ) : props.href.startsWith('#') || props.href.startsWith('http') ? (
```
It could also be `https`

Reply:
I know, but if the url starts with `https`, that is composed with `http` so the result will be `true`

![Screenshot (281)](https://user-images.githubusercontent.com/72418962/119535506-114c9200-bd56-11eb-83af-11c3be51cf38.png)